src/buildercore/project/files.py

```diff
@@ -18,6 +18,7 @@ def update_project_file(path, value, pdata, project_file):
     utils.updatein(pdata, path, value, create=True)
     return pdata
 
+# TODO: deletion candidate
 @testme
```

Comment:
I agree, we should not use a file both as a source and of data as a target of generation

Reply:
it's from the earlier days of the builder where the projects file could be post-processed and updated. Comments weren't retained but I could always go back through the diff and restore them.

I'm adding the comments as I see a code removal branch in builder's near future to trim it down a bit

Reply:
I see that thanks to DST-magic we cross over a bit now

Reply:
It's also 7pm here and I can't seem to stop working today ;)

Stopping now. I need food.